js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+var $urlInputBox = document.querySelector('#photoUrl');
+var $photoUrl = document.querySelector('.img-box');
+var $entryForm = document.querySelector('#entry-form');
+
+function handleInput(event) {
+  if (event.target.matches('#photoUrl') === true) {
```

Comment:
You applied this event handler to this specific element and no others, so this conditional isn't necessary. Additionally, event handler functions do not need a return statement, since you will not ever be able to access that value.

Reply:
You only need the line to set the `src` attribute